build-tests/api-extractor-test-01/etc/api-extractor-test-01.api.ts

```diff
@@ -7,6 +7,11 @@ class AmbientConsumer {
   localTypings(): IAmbientInterfaceExample;
 }
 
+// @public
+class DecoratorTest {
+  test(): void;
```

Comment:
> test(): void; [](start = 1, length = 14)

FYI the reason the "@virtual" decorator isn't coming through is that it doesn't get emitted into the *.d.ts file. This is actually an unforeseen usability problem with these decorators, since it means they won't show up in IntelliSense or F12 'go-to-definition".

I asked the compiler people about it, and there isn't a straightforward way to emit decorators in *.d.ts files, since they are executable code (and experimental).

I'm thinking in an upcoming PR we should convert these decorators to AEDoc tags.

@mpasarin FYI